pydoctor/test/test_epydoc2stan.py

```diff
@@ -128,6 +159,83 @@ def f(a):
     classic_fmt = docstring2html(classic_mod.contents['f'])
     assert annotation_fmt == classic_fmt
 
+def test_func_param_duplicate(capsys: CapSys) -> None:
+    """Warn when the same parameter is documented more than once."""
```

Comment:
Do we already have a test when the same parameter has multiple `@type` definitions?